package.json

```diff
@@ -16,25 +16,25 @@
   "dependencies": {
     "@typescript-eslint/eslint-plugin": "^5.26.0",
     "eslint-plugin-simple-import-sort": "^7.0.0",
```

Comment:
Should we move these to dev dependencies?

Reply:
Yes sounds good